environment.yml

```diff
@@ -1,15 +1,18 @@
 name: pcp-env
 channels:
  - conda-forge
- - mathieu
+ #- mathieu
```

Comment:
Feel free to remove altogether since `pdal-python` is now available through conda-forge.